docs/guides/joint_fabric_guide.md

```
@@ -280,6 +305,16 @@ On the Ecosystem A Joint Fabric Controller application
 pairing code 10 [manual pairing code] --jcm true
 ```
 
+After successful JCM commissioning, check for the following logs on Ecosystem
+A's jf-admin-app to confirm the cross-signing flow completed:
+
+```
```

markdownlint-cli2 (0.23.2)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)